packages/testing/src/normalize.ts

```diff
@@ -37,6 +37,10 @@ const NORMALIZE_REGEXPS = [
     /(^|[ "'(=])((?:\.{0,2}|([A-Z]:)|file:\/\/)(\/[^ "')\n]+))/gm,
 
     (_, prefix, pathMatch, winDrive, pathTrail) => {
+      if (pathMatch.includes('/$netlify-snapshot-preserve/')) {
```

Comment:
Without this, all paths would get normalised to `/external/path` and the snapshot would be useless in telling us which redirects take precedence. This acts as an escape hatch for the path normalisation.

Yes, this is weird. But so is the whole normalisation logic we have in snapshot tests, unfortunately.